xzre.h

```diff
@@ -708,31 +708,39 @@ assert_offset(global_context_t, shift_operations, 0x141);
 assert_offset(global_context_t, num_shifted_bits, 0x160);
 static_assert(sizeof(global_context_t) == 0x168);
 
-typedef struct __attribute__((packed)) {
+typedef struct __attribute__((packed)) backdoor_shared_globals {
 	PADDING(0x10);
-	global_context_t *globals;
+	global_context_t **globals;
```

Comment:
took me a while to realise this but after tracing the assignments this is a pointer to a pointer